libphobos/m4/druntime/libraries.m4

```diff
@@ -16,6 +40,17 @@ AC_ARG_ENABLE(thread-lib,
 ])
 
 
+# DRUNTIME_LIBRARIES_TIME
+# -----------------------
++# Check whether -lrt is required for sched_yield, nanosleep, or clock_gettime.
+AC_DEFUN([DRUNTIME_LIBRARIES_TIME],
+[
+AC_SEARCH_LIBS([sched_yield], [rt])
+AC_SEARCH_LIBS([nanosleep], [rt])
+AC_SEARCH_LIBS([clock_gettime], [rt])
+])
+
```

Comment:
> As one possible solution: Could we dump the required libraries to a file and parse that file in the driver?

Maybe could draw some inspiration from Ada.

https://gcc.gnu.org/onlinedocs/gcc-6.2.0/gnat_ugn/The-Ada-Library-Information-Files.html#The-Ada-Library-Information-Files